Sources/Clappr_tvOS/Classes/Base/Player.swift

```diff
@@ -313,6 +315,20 @@ open class Player: AVPlayerViewController {
         baseObject.trigger(event.rawValue, userInfo: userInfo)
     }
 
+    open func enterChromelessMode() {
+        isChromelessModeEnabled = true
+        showsPlaybackControls = false
+        view.isUserInteractionEnabled = false
+        activeContainer?.trigger(InternalEvent.didEnterChromelessMode.rawValue)
+    }
+
+    open func exitChromelessMode() {
+        isChromelessModeEnabled = false
+        showsPlaybackControls = true
+        view.isUserInteractionEnabled = true
+        activeContainer?.trigger(InternalEvent.didExitChromelessMode.rawValue)
+    }
```

Comment:
I recommend making this functions private and keeping all chromeless mode control on a property. 
This is closer to the chromeless mode control on Clappr and Clappr-Android: a dynamic option.

Reply:
Done